proto/storage/report_configuration.proto

```diff
@@ -79,3 +79,15 @@ message ResourceScope {
     string collection_id = 1; // @gotags: search:"Collection ID"
   }
 }
+
+//filter for ondemand reports
+message OndemandVulnerabilityReportFilters {
+  enum ImageType {
+    DEPLOYED = 0;
+    WATCHED = 1;
+  }
+  repeated ImageType image_types = 2;
+  bool include_nvd_cvss = 3;
+  bool include_epss_probability = 4;
+  string query = 5;
```

Comment:
Out of curiosity, what does this string look like?  Do you have an example?

Reply:
this is query string sent by UI - "SEVERITY:Critical,Important+Image CVE Count:>0+Platform Component:false+CVE:"